packages/core/app-extensions/src/form/formModifier.js

```diff
@@ -5,6 +5,70 @@ export const ACTION_SAVE_ID = 'save'
 export const ACTION_GROUP_OUTPUT_ID = 'output'
 export const ACTION_GROUP_ACTIONS_ID = 'actions'
 
+export const removeBoxes = (formDefinition, boxIds) => ({
+  ...formDefinition,
+  children: formDefinition.children
+    .filter(item => item.componentType !== 'layout' || !boxIds.includes(item.id))
+    .map(item => {
+      if (item.componentType === 'layout') {
+        return removeBoxes(item, boxIds)
+      }
+
+      return item
+    })
+})
+
+export const removeActions = (formDefinition, actionIds) => ({
+  ...formDefinition,
+  children: formDefinition.children.map(rootItem => {
+    if (rootItem.id === MAIN_ACTION_BAR_ID) {
+      return {
+        ...rootItem,
+        children: rootItem.children
+          .map(group => {
+            if (group.componentType === 'action-group') {
```

Comment:
Can we move those static strings to a constant (like we have it for MAIN_ACTION_BAR_ID)?